android/local.properties

```diff
@@ -1,8 +0,0 @@
-## This file must *NOT* be checked into Version Control Systems,
```

Comment:
This file should be in `.gitignore` anyway, in case anyone needs to override local gradle configs. Sample `.gitignore` for comparison: https://github.com/github/gitignore/blob/master/Android.gitignore (we probably don't need all of this).

Reply:
done (more can be added in the future for ignored files)